presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientConfig.java

```diff
@@ -167,6 +167,7 @@ public void testExplicitPropertyMappings()
                 .put("hive.s3.pin-client-to-current-region", "true")
                 .put("hive.parquet-predicate-pushdown.enabled", "true")
                 .put("hive.parquet-optimized-reader.enabled", "true")
+                .put("hive.orc.bloomfilters", "false")
```

Comment:
it's a test for explicitly changing the value, the default is true, so setting it to true again makes the test fail